### PR TITLE
HAWKULAR-1040 Issue with URL trigger definition

### PR DESCRIPTION
**Pull request by jshaughn**
- handle the fact that the actual URL is not supplied in
  the resource create event published by inventory.

### CODE REVIEW

File: modules/hawkular-api-parent/hawkular-listener/src/main/java/org/hawkular/listener/bus/InventoryEventListener.java [hunk 1]
```diff
@@ -696,14 +696,10 @@ private void handleResourceEvent(ResourceEvent event) {
...
-                    String feedId = r.getPath().ids().getFeedId();
-                    String url = event.getHeaders().get("url");
-                    //String resourceId = String.valueOf(MessageDigest.getInstance("MD5").digest(
-                    //        r.getId().getBytes("UTF-8")));
+                    // TODO: use the URL as needed, not the hash if and when we can get the actual URL from the event
                     String resourceId = r.getId();
-                    //String qualifiedResourceId = feedId + "/" + resourceId;
                     Map<String, String> memberContext = new HashMap<>(2);
-                    memberContext.put("resourceName", url);
+                    memberContext.put("resourceName", "URL [" + resourceId + "]"); // Should be the URL
```
Comment (lucasponce) [inline modules/hawkular-api-parent/hawkular-listener/src/main/java/org/hawkular/listener/bus/InventoryEventListener.java]: url == resourceId is confirmed in all cases ?

File: modules/hawkular-api-parent/hawkular-listener/src/main/java/org/hawkular/listener/bus/InventoryEventListener.java [hunk 2]
```diff
@@ -711,8 +707,7 @@ private void handleResourceEvent(ResourceEvent event) {
                     // Response Time
                     String groupTriggerId = "URL_Response";
                     String memberId = groupTriggerId + "_" + resourceId;
-                    String memberName = "URL Response [" + url + "]";
```
Comment (lucasponce) [inline modules/hawkular-api-parent/hawkular-listener/src/main/java/org/hawkular/listener/bus/InventoryEventListener.java]: So, perhaps a memberName should be good, to not leave this trigger without name.

Comment (jshaughn) [inline modules/hawkular-api-parent/hawkular-listener/src/main/java/org/hawkular/listener/bus/InventoryEventListener.java]: It defaults to the group trigger name, which is currently the policy for all of the member triggers.

File: modules/hawkular-api-parent/hawkular-listener/src/main/java/org/hawkular/listener/bus/InventoryEventListener.java (hunk 1 above)
Comment (jshaughn) [inline modules/hawkular-api-parent/hawkular-listener/src/main/java/org/hawkular/listener/bus/InventoryEventListener.java]: The actual URL is not supplied by inventory, so using the resourceId is a workaround until such time that we have the URL.

**Comment by lucasponce** [comment]
LGTM

**Pull request merged**